applications/the-coffee-bar-frontend/public/index.html

```diff
@@ -35,6 +35,7 @@
           collectionSourceUrl: "%REACT_APP_COLLECTION_SOURCE_URL%",
           serviceName: "%REACT_APP_SERVICE_NAME%",
           applicationName: "%REACT_APP_APPLICATION_NAME%",
+          deploymentEnvironment: %REACT_APP_ENVIRONMENT_NAME%,
```

Comment:
Please add default value for `REACT_APP_ENVIRONMENT_NAME` env var in deployments (docker-compose and helm values.yaml).

Reply:
done